ArchiSteamFarm/www/css/app.css

```diff
@@ -12,7 +12,7 @@
 }
 
 .box-content-log {
-    font-family: "lucida console";
+    font-family: "Courier";
```

Comment:
Courier looks terrible on my machine. Perhaps it'd be better if we used a [generic font family](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#<generic-name>) in this case. [normalize.css](https://github.com/necolas/normalize.css/blob/73b6b0c7e8690ab5005bca9d7e13d3fb319c98ac/normalize.css#L90-L93) uses `font-family: monospace, monospace` ([explanation](https://stackoverflow.com/a/38781315)). From my tests the chosen font looked perfect on all major browsers on Windows, Linux and Android. See for yourself

Reply:
Added it! Looks good on Windows and Mac. Cant test Linux but if you say it looks good I believe you. Thank you!

Reply:
Nice, thank you. It really [does look much better](https://i.imgur.com/Tx6PEEk.png) than [Courier](https://i.imgur.com/w6Co19K.png)!